docs/guides-use-frame.md

```diff
@@ -4,10 +4,106 @@ title: How to use Frame
 sidebar_label: How to use Frame
 ---
 
-[//]: # "TODO: create guide"
-
-This guide help you get started with Frame to interact with your DAO. If you don't have a DAO yet create one following [your first aragon app tutorial](tutorial.md).
+This guide will help you to get started with Frame and interact with a DAO using a hardware wallet.
 
 ## Setup
 
-Start by downloading Frame from the official website: https://frame.sh.
+Start by downloading Frame from the official website: https://frame.sh. After installation Frame stays quietly in your menu bar until it's needed.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-intro.gif">
+</p>
+
+To take full advantage of Frame we recommend you have the `aragonCLI` installed as well. If you already have it, skip to the next section, otherwise ran:
+
+```sh
+npm install -g @aragon/cli
+```
+
+> You need a Ledger or Trezor wallet to use Frame.
+
+## Configure you Device
+
+#### Ledger
+
+On Ledger devices insert your pin on the device to unlock it and then open Frame.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-ledger.gif">
+</p>
+
+#### Trezor
+
+On Trezor devices insert your pin directly on Frame.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-trezor.gif">
+</p>
+
+You are ready to select the network and account you would will use.
+
+#### Network
+
+To choose the appropiate network select the button on the uper right corner (the one with the three arrows). This page is the app menu, here you can configure general settings for Frame. We will focus on the Connection option. Right now the default is Rinkeby but you can aslo choose Mainnet, Ropsten and Kovan. For this tutorial we will select Rinkeby.
```

Comment:
```suggestion
To choose the appropriate network, select the button on the upper right corner (the three arrows). This will take you to Frame's menu, where you can configure its general settings. We will focus on the "Connection" option. Right now the default is Rinkeby but you can also choose Mainnet, Ropsten or Kovan.

For this tutorial we will select Rinkeby.
```